backend/src/dsErrors.js

```diff
@@ -0,0 +1,23 @@
+export class EmailAlreadyExistsError extends Error {
```

Comment:
Like the idea of gathering error types in a file.

Reply:
:+1: